src/stdatamodels/jwst/datamodels/level1b.py

```diff
@@ -27,3 +52,9 @@ class Level1bModel(JwstDataModel):
 
     """
     schema_url = "http://stsci.edu/schemas/jwst_datamodel/level1b.schema"
+
+    def __init__(self, init=None, **kwargs):
+        if isinstance(init, fits.HDUList):
+            init = _migrate_moving_target_table(init)
+
+        super(Level1bModel, self).__init__(init=init, **kwargs)
```

Comment:
Similar to https://github.com/spacetelescope/stdatamodels/pull/183 would you add a call to `_migrate_moving_target_table` to the `_migrate_hdulist` method for this subclass? I think this should do it.
```suggestion
        super(Level1bModel, self).__init__(init=init, **kwargs)

    def _migrate_hdulist(self, hdulist):
        return _migrate_moving_target_table(hdulist)
```

Reply:
Happy to do that, but I didn't see it being called. Should the call in `__init__` be to `_migrate_hdulist` instead of `_migrate_moving_target_table`?

Reply:
It's called during the `super` call when init is:
- a [hdulist](https://github.com/spacetelescope/stdatamodels/blob/234413f1209c50b9b103239279c813fa678dec81/src/stdatamodels/model_base.py#L217) this overlaps with what's currently in this PR (and what was in #183). I don't recall why it was added separately in the subclass `__init__` as the migration should be triggered in the `super` call for an hdulist.
- a [filename](https://github.com/spacetelescope/stdatamodels/blob/234413f1209c50b9b103239279c813fa678dec81/src/stdatamodels/model_base.py#L231) this is the main reason to add `_migrate_hdulist` since otherwise a call to `Level1bModel(filename)` won't trigger the migration.

Maybe the thing to do would be to:
- add a unit test
- drop the subclass `__init__`
- implement `_migrate_hdulist`

and see if that fixes the problem (or perhaps reveals why the previous PR also added the migration to the subclass).